### DISCUSSION TITLE
UniGetUI 3.1.4 beta 1

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2992] [Announcements]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.4-beta1/total?style=for-the-badge&color=red)

> [!CAUTION]
> A new autoupdater has been introduced. It is **highly recommended** to enable "prerelease updates" from UniGetUI Settings once you have the new build installed. This will allow you both to test the new autoupdater and to automatically receive new prerelease builds when available.

# Relevant changes
 - Package lists will show icons for packages who have them available.
 - This is the first release that uses [UniGetUI Elevator](https://github.com/marticliment/GSudo-for-UniGetUI/) instead of gsudo
 - Add support for vcpkg
 - Important improvements to the autoupdater (please read above)
 - Improvements to ignored packages
 - Improvements to page navigation and keyboard shortcuts
 - Improvements to package manager error handling
 - Fixed _the_ issue with PowerShellGet 1.0 when updating packages

SHA256: `B08D464CF5385120A96A99C9BD0EA41F2B7626C5AE4696F127695598687F921D`

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/UniGetUI/releases/tag/3.1.4-beta1'>UniGetUI 3.1.4 beta 1</a>.</em>

**Comment by BrianG61UK** [comment]
On one of my PCs this prompted me saying it needed to install parse_pip_search (yes, with underscores!) every time it started even though parse-pip-search 0.0.1 is already installed. I had to tick the box to tell it not to ask again!

**Reply by marticliment** [reply]
What happens if you run the command `parse_pip_search.exe` on a command prompt?

**Reply by BrianG61UK** [reply]
C:\>parse_pip_search.exe
usage: pip_search [-h] [-s [{name,version,released}]] [--version] [--date_format [DATE_FORMAT]] [query ...]

Search for packages on PyPI

positional arguments:
  query                 terms to search pypi.org package repository

options:
  -h, --help            show this help message and exit
  -s [{name,version,released}], --sort [{name,version,released}]
                        sort results by package name, version or release date (default: name)
  --version             show program's version number and exit
  --date_format [DATE_FORMAT]
                        format for release date, (default: %d-%m-%Y)

C:\>parse_pip_search.exe --version
pip_search 0.0.1

C:\>

**Reply by marticliment** [reply]
This is weird. It should get recognized then...

I will investigate the issue

**Reply by BrianG61UK** [reply]
~~I think it's fixed itself now. It doesn't seem to be showing in manage ignored updates. I can't remember if I removed it from there or what happened. Sorry. Maybe it was an error in the data at PyPI ?~~ I think I must have forgotten what I did. See next comment:

**Reply by BrianG61UK** [reply]
It happened again when I updated to beta2. I had to tick the box to ignore the error.

**Reply by marticliment** [reply]
Yes, I am afraid this beta didn't contain a fix for this.

Were you able to update correctly through the built in updater or you installed the beta manually?

**Reply by BrianG61UK** [reply]
I'm sorry, I don't remember. :-(

**Comment by divariv** [comment]
I was also prompted that parse-pip-search was missing.
Installed immediately, but I didn't close the app. I was pulling my hair out trying to find the prerelease update toggle in the settings, only to eventually realize that it doesn't appear in the ui until restarting the app after that package is installed. Once I re-opened, it was right where I expected!
